pkg/common/proto/agent.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
```

Comment:
Stylewise field names should be snake_case: https://protobuf.dev/programming-guides/proto3/#field-labels

The code gen will convert those into appropriate styles for each language.